### PR TITLE
19 migrate cfrs from forests

### PR DESCRIPTION
**Pull request by peacog**

### CODE REVIEW

File: docker/drupal/settings.overrides.php [hunk 1]
```diff
@@ -16,6 +16,18 @@
   'collation' => 'utf8mb4_general_ci',
 ];
 
+/**
+ * Temporary migration source DB.
+ */
+$databases['migrate']['default'] = [
```
Comment (pcambra) [inline docker/drupal/settings.overrides.php:22]: Is this file for staging/prod?

File: web/modules/custom/brms_common/brms_common.deploy.php [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+ * Deploy functions run after drush config:import
+ */
+
+use Drupal\migrate\MigrateExecutable;
+use Drupal\migrate\MigrateMessage;
+
+/**
+ * Initialise taxonomy terms.
+ */
+function brms_common_deploy_001(&$sandbox = NULL) {
```
Comment (pcambra) [inline web/modules/custom/brms_common/brms_common.deploy.php:14]: I like the idea of numbering the deploys but probably should be better to do `brms_common_deploy_001_short_description`? I'll add an adr proposal

File: web/modules/custom/brms_migrate/migrations/brms_migrate_cfrs.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+langcode: en
```
Comment (pcambra) [inline web/modules/custom/brms_migrate/migrations/brms_migrate_cfrs.yml:1]: I've always wondered how the "migrations" folder works, do they get picked up automatically?

Comment (pcambra) [reply web/modules/custom/brms_migrate/migrations/brms_migrate_cfrs.yml:1]: I've used migrate_manifest for this in the past

File: web/modules/custom/brms_migrate/src/Plugin/migrate/source/Asset.php [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+namespace Drupal\brms_migrate\Plugin\migrate\source;
+
+use Drupal\migrate\Plugin\migrate\source\SqlBase;
+
+/**
+ * Asset source from database.
+ *
+ * @MigrateSource(
+ *   id = "farmos_asset",
```
Comment (pcambra) [inline web/modules/custom/brms_migrate/src/Plugin/migrate/source/Asset.php:11]: Is this coming from farmos migration?

File: docker/drupal/settings.overrides.php (hunk 1 above)
Comment (peacog) [reply docker/drupal/settings.overrides.php:22]: Yes. Those are the values that have been configured by SBIT on staging

File: web/modules/custom/brms_migrate/migrations/brms_migrate_cfrs.yml (hunk 3 above)
Comment (peacog) [reply web/modules/custom/brms_migrate/migrations/brms_migrate_cfrs.yml:1]: They get picked up automatically and the beauty of them is you don't have to reimport config every time you make a change to the definition. Very useful during development. One downside is that they don't show up in the list of migrations in drupal UI so you can't run them from there. `drush migrate:status` does list them.

File: web/modules/custom/brms_migrate/src/Plugin/migrate/source/Asset.php (hunk 4 above)
Comment (peacog) [reply web/modules/custom/brms_migrate/src/Plugin/migrate/source/Asset.php:11]: No, it's defined here in brms_migrate

Comment (pcambra) [reply web/modules/custom/brms_migrate/src/Plugin/migrate/source/Asset.php:11]: Yes, but did we get it from somewhere else or it is developed ad-hoc

Comment (peacog) [reply web/modules/custom/brms_migrate/src/Plugin/migrate/source/Asset.php:11]: It's ad-hoc, taking inspiration from Migrate Plus examples

**Pull request merged**